server/models/User.ts

```diff
@@ -3,7 +3,6 @@ import mongoose, { Schema } from 'mongoose';
 const userSchema = new Schema({
     email: { type: String, required: true, unique: true },
     password: { type: String, required: true, unique: true },
-    projects: { type: Array, required: true },
```

Comment:
Similar to above, shouldn't a user have a projects array? Maybe I'm missing something.

Reply:
Projects store their associated userID